diff --git a/core/qcom_utils.mk b/core/qcom_utils.mk
index 1177ad4..7dbf13f 100755
--- a/core/qcom_utils.mk
+++ b/core/qcom_utils.mk
@@ -1,5 +1,7 @@
 # Board platforms lists to be used for
 # TARGET_BOARD_PLATFORM specific featurization
+QCOM_BOARD_PLATFORMS += msm7x27
+QCOM_BOARD_PLATFORMS += msm7x27a
 QCOM_BOARD_PLATFORMS += msm7x30
 QCOM_BOARD_PLATFORMS += msm8226
 QCOM_BOARD_PLATFORMS += msm8610
